apps/gateway/src/lib/costs.spec.ts

```
@@ -356,6 +356,139 @@ describe("calculateCosts", () => {
+			undefined,
+			null,
+			1, // 1 output image
+			"0.5K",
+			0,
+		);
+
+		expect(result.imageOutputTokens).toBe(747); // 1 * 747
+		expect(result.imageOutputCost).toBeCloseTo(747 * (60 / 1e6)); // 747 * $60/1M
+		const textTokens = 800 - 747; // 53 text tokens
+		expect(result.outputCost).toBeCloseTo(
+			textTokens * (1.5 / 1e6) + 747 * (60 / 1e6),
```

ESLint

[error] Unexpected mix of '*' and '+'. Use parentheses to clarify the intended order of operations.

(no-mixed-operators)